src/AspNet.Security.OAuth.Vkontakte/README.md

```diff
@@ -0,0 +1,22 @@
+AspNet.Security.OAuth.Vkontakte
```

Comment:
Please remove this file.

I'm not opposed to having a README in subfolders but we'd have to do that for all the providers, for consistency.